components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -2901,7 +2904,7 @@ void p3_main_part2_f(
     mu_r_d              (temp_d[50]),
     lamr_d              (temp_d[51]),
     logn0r_d            (temp_d[52]),
-    cmeiout_d           (temp_d[53]),
+    qv2qi_depos_tend_d           (temp_d[53]),
```

Comment:
the white-space alignment is matched for `th_atm_d` on line 2878 but here it isn't...